_docs/Qabel-Protocol-Box.md

```diff
@@ -5,22 +5,20 @@ title: "Protocol: box"
 
 ## Abstract
 
-A set of protocols to store files and folders on a VOLUME that is on AWS S3 and managed by a third party who doesn't need to be trusted.
+A set of protocols to store files and folders on a VOLUME that is (currently) stored on AWS S3 and managed by a third party who doesn't need to be trusted.
 
 ## Used services
 
-Qabel Box uses an Accounting server which controls the access to AWS S3. Every client who needs write access has to be authenticated by the server and then receives a set of credentials for direct access to the VOLUME.
-Qabel Box also directly uses AWS S3 to store the blocks and metadata.
+Qabel Box uses an Accounting server that controls the access to the Qabel Block server which directly accesses the files on AWS S3. Every client which needs write access has to be authenticated by the Accounting server and then receives an authentication token for this indirect access to the VOLUME.
+Qabel Box also uses the Block server to store the blocks and metadata.
 
 ## Accounting server
 
-The accounting server controls write access to the S3 bucket. Registered users (not identities!) can request temporary credentials for AWS.
-All data is sent as JSON and UTF-8.
-
-Data types are defined [here](../Qabel-Client-Local-Data#data-types).
+The accounting server controls write access to the Block server. Registered users ( = accounts != identities) can request temporary authentication tokens for the access.
+All data is sent as JSON and UTF-8. All data types are defined [here](../Qabel-Client-Local-Data#data-types).
 
 ### Required permissions for the AWS user
-The accounting server uses an AWS user to create the temporary credentials. He must have the following permissions:
+The accounting server uses an AWS user to create the temporary tokens. He must have the following permissions:
```

Comment:
The accounting server doesn't have AWS credentials, only the block server (which is an implementation detail)